bottle_app.py

```diff
@@ -80,14 +82,27 @@ def reports():
 def search():
     return bottle.jinja2_template('search.html', template_lookup=[TEMPLATE_DIR])
 
+@bottle.route('/index.tsv')
```

Comment:
@simsong I'm not familiar with Bottle.  Is this path cached with an expiry date/expiry trigger, or does it run the database query on every access?

Reply:
It will give us a heavy database access on every hit, but this is no different than the search API.
If this becomes an issue, we can cache to the file system or to S3, and automatically re-generate if it is more than  1 minute old.

Reply:
Yep, sounds good to me!